changelog.d/pr-6184.bugfix

```diff
@@ -0,0 +1 @@
+Fixed home screen shrinking too much on opening the keyboard.
```

Comment:
Will this differently named changelog be picked up?

Reply:
Yep, it's part of our normal naming scheme - https://github.com/matrix-org/matrix-ios-sdk/blob/develop/CONTRIBUTING.md#changelog -> `pr-[PRNumber].type for a PR with no related issue`